index.html

```diff
@@ -65,5 +69,23 @@ <h2>Messages</h2>
     <footer>
     </footer>
     <script src="js/index.js"></script>
+    <script>
```

Comment:
I think you want to move this to your `js/index.js` file above.

Reply:
Thank you for your review, I moved the code to js file.